[Cosmos] Change default consistency level to the database account consistency level (#22792)
Azure/azure-sdk-for-python: sdk/cosmos/azure-cosmos/README.md

### Using the asynchronous client (Preview)

The asynchronous cosmos client is a separate client that looks and works in a similar fashion to the existing synchronous client. However, the async client needs to be imported separately and its methods need to be used with the async/await keywords.

> Due to its asynchronous nature, it also needs to be warmed up and then closed down when used. The example below shows how by using the client's __aenter__() and close() methods.

Reviewer: ```suggestion
The Async client needs to be initialized and closed after usage. The example below shows how to do so by using the client's __aenter__() and close() methods.
```

    await client.close() # the async client must be closed manually if it's not initialized in a with statement
```

> Instead of manually opening and closing the client, you can and should use the `async with` keywords. This creates a context manager that will initialize and later close the client once you're out of the statement. The example below shows how to do so.

Reviewer 2: ```suggestion
Instead of manually opening and closing the client, it is highly recommended to use the `async with` keywords. This creates a context manager that will initialize and later close the client once you're out of the statement. The example below shows how to do so.
```

> However, instead of taking care of it manually with those extra lines, you can use the `async with` keywords. This creates a context manager that will warm up, initialize and later clean up and close the client once you're out of the statement. The example below shows how to start the client this way.

Reviewer: ```suggestion
Instead of manually opening and closing the client, you can use the `async with` keywords. This creates a context manager that will initialize and later close the client once you're out of the statement. The example below shows how to do so.
```

DATABASE_NAME = 'testDatabase'
CONTAINER_NAME = 'products'

async def create_products():

>     async with CosmosClient(URL, credential=KEY) as client: # the with statement will automatically warm up, initialize,# and close the async client

Reviewer: ```suggestion
    async with CosmosClient(URL, credential=KEY) as client: # the with statement will automatically initialize and close the async client
```
